src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf

```diff
@@ -3455,7 +3455,7 @@
         </context-group>
       </trans-unit><trans-unit id="f7a13e4807afa6a9eda3db57a0a2a110dbae8ad7" datatype="html">
         <source><x id="ICU" equiv-text="{mode, select, edit {...} other {...}}"/> the pool
-          mirror peer attributes for pool <x id="START_TAG_KBD" ctype="x-kbd" equiv-text="&lt;kbd&gt;"/><x id="INTERPOLATION" equiv-text="{{ poolName }}"/><x id="CLOSE_TAG_KBD" ctype="x-kbd" equiv-text="&lt;/kbd&gt;"/> and click 
+          mirror peer attributes for pool <x id="START_TAG_KBD" ctype="x-kbd" equiv-text="&lt;kbd&gt;"/><x id="INTERPOLATION" equiv-text="{{ poolName }}"/><x id="CLOSE_TAG_KBD" ctype="x-kbd" equiv-text="&lt;/kbd&gt;"/> and click
```

Comment:
Just out of curiosity, is this change manual or some editor has removed that trailing whitespace?

Reply:
Gah, didn't notice that. Must have been my editor (MS Visual Studio Code)...
Hmm. Let me try to fix this.

Reply:
Should be fixed now.

Reply:
+1

Reply:
Meh, manually changing these strings didn't work, I'm getting `Please run 'npm run i18n' and commit the modified 'messages.xlf' file.`. Redoing this once again...